src/solver/optim-model-filler/ReadLinearConstraintVisitor.cpp

```diff
@@ -104,82 +104,82 @@ static std::invalid_argument IllegalNodeException()
     return std::invalid_argument("Root node of a constraint must be a comparator.");
 }
 
-std::vector<LinearConstraint> ReadLinearConstraintVisitor::visit(const SumNode* sum_node)
+std::vector<LinearConstraint> ReadLinearConstraintVisitor::visit(const SumNode*)
```

Comment:
Class `Node` exposes a `std::string name() const` method, it may be useful to include that name into the `IllegalNodeException` for debugging purposes.